src/TeamContributionCalendar/TeamContributionCalendar.js

```diff
@@ -140,10 +143,19 @@ export default class TeamContributionCalendar {
       if (data.error) {
         console.error(data.errorMessage);
       } else {
-        this.processGitHubCalendar(data.parsedCalendar, {
-          start: user.from,
-          end: user.to
-        });
+        const timeframe = { start: user.from, end: user.to };
+        // This removes the "noise" from the calendar and sorts the days out falling out of the specified timeframe.
```

Comment:
"sorts the days out falling out of the specified timeframe" does this mean "removes days falling out of the specified timeframe"?

Reply:
I found this a bit weird :D Could you use the second form?

Reply:
Yea, sure.

https://github.com/c-hive/team-contribution-calendar/pull/70#discussion_r408411065

We may be able to get rid of these comments if there's a way to wrap the logic with a function.

Reply:
Obsolete.